pkg/apis/awsprovider/v1alpha1/register.go

```diff
@@ -29,7 +29,6 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/json"
-
```

Comment:
i think this is new gofmt behavior? i don't know why this is happening

Reply:
Don't worry about it atm. #290 should give us a canonical formatting.